pom.xml

```diff
@@ -696,6 +696,24 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>io.confluent</groupId>
+                <artifactId>resolver-maven-plugin</artifactId>
+                <version>0.1.0</version>
+                <configuration>
+                    <groupId>org.apache.kafka</groupId>
+                    <artifactId>kafka-clients</artifactId>
+                    <versionRange>[6.0.0-0, 6.0.1-0)</versionRange>
```

Comment:
I need to make this version range point to a property so we can easily update it during branching